redisvl/providers/__init__.py

```diff
@@ -0,0 +1,9 @@
+
+from redisvl.providers.openai import OpenAIProvider
```

Comment:
should we consider organizing it one layer deeper, under `embeddings`? that way its clear whether its for Embedding creation or something else (future proofing)

Reply:
That's a long import but I see what you mean. A couple options

```python
from redisvl.api.embedding.openai
from redisvl.embedding.openai import OpenAIEmbedding
from redisvl.model.embedding.openai import ...

```

others?